itez/beneficiary/migrations/0008_beneficiary.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 3.1.13 on 2021-11-04 09:29
+
+from django.db import migrations, models
+import uuid
```

Comment:
built in python imports should always come first before django imports

Reply:
This is in the migration file that was generated, I have no control over how imports are made